mne/preprocessing/ica.py

```diff
@@ -144,36 +161,57 @@ class ICA(ContainsMixin):
     Parameters
     ----------
     n_components : int | float | None
-        The number of components used for ICA decomposition. If int, it must be
-        smaller than `max_pca_components`. If None, all PCA components will
-        be used. If float between 0 and 1, components will be selected by the
-        cumulative percentage of explained variance.
+        Controls the number of PCA components from the pre-ICA PCA entering the
+        ICA decomposition in the `fit()` method.
+        If None (default), all PCA components will be used
+        (== `max_pca_components`).
+        If int, must be <= `max_pca_components`.
+        If float between 0 and 1, the number of components selected matches the
+        number of components with a cumulative explained variance below
+        `n_components` (<1> resulting in `max_pca_components`).
```

Comment:
<1>? I don't think we have ever used this notation, you can just do "1 resulting in" or "a value of 1 resulting in"